### DISCUSSION TITLE
How to use Nuxt 3 with Deno?

### DISCUSSION THREAD

**Discussion by fleximus** [nuxt/nuxt#17474] [Questions] [answered]
Hi,

Nuxt 3 was announced to natively support Deno. Neither I was able to find a piece of documentation how to run Nuxt 3 on Deno, nor was I able to figure that out by myself.

Now I'm kindly asking for someone showing off how to do that by either sample code, describing the needed steps, writing a blog post, and so on.

Many thanks and keep being a good community. 💚

**Comment by pi0** [comment, marked as answer]
Hi, dear @fleximus. Nuxt does not natively support deno. It works with Node.js. but you can build and deploy a Nuxt 3 application to [Deno Deploy](https://deno.com/deploy) soon with a [Nitro](https://nitro.unjs.io) preset.

**Reply by TheAlexLichter** [reply]
(Addition: For the Deno Deploy preset, there is a trackable issue @ https://github.com/unjs/nitro/issues/53)

**Reply by nizasichi** [reply]
@TheAlexLichter , is there any hope that this will become a possibility?

**Reply by TheAlexLichter** [reply]
@devniza

* [Deno Deploy](https://nitro.build/deploy/providers/deno-deploy) is already possible 👍🏻
* Deno Serve [works too](https://nitro.build/deploy/runtimes/deno)
